test/NuGet.Core.Tests/NuGet.Common.Test/DateTimeUtilityCultureInvariantTests.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Xunit;
+
+namespace NuGet.Common.Test
+{
+    [Collection(LocalizedTestCollection.TestName)]
+    public class DateTimeUtilityCultureInvariantTests
+    {
+        public static IEnumerable<object[]> GetData()
+        {
+            return new[]
+            {
+                new object[] { "1.23 sec", TimeSpan.FromSeconds(1.23d) },
```

Comment:
Just checking my understanding here, but is this test relying on the host machine to be set to "en-US" ? Or does it not matter which culture is used because "ms" and "hr" and "min" are cross-compatible?

ie, if date format (mm/dd/yyyy versus dd/mm/yyyy) were being tested, then I don't see how the `expected` test data could be used.

Reply:
Ok, yes it seems that non-country-specific English is the default. Maybe the test could be named better? It wasn't clear to me what we're testing by "Succeeds". Guess we're testing that the utility treats invariant culture the same as specific cultures?

Reply:
Ok. Renamed and tested only time number. Removed localized time unit testing.